browser/extensions/api/brave_shields_api.cc

```diff
@@ -66,31 +66,31 @@ BraveShieldsUrlCosmeticResourcesFunction::Run() {
 
 std::unique_ptr<base::ListValue> BraveShieldsUrlCosmeticResourcesFunction::
     GetUrlCosmeticResourcesOnTaskRunner(const std::string& url) {
-  base::Optional<base::Value> resources = g_brave_browser_process->
-      ad_block_service()->UrlCosmeticResources(url);
+  // base::Optional<base::Value> resources = g_brave_browser_process->
```

Comment:
why are these all commented out?

Reply:
as @antonok-edm pointed out above, we don't need that anymore as we want to get rid of the extension